alpaca/rest.go

```diff
@@ -314,6 +315,8 @@ type CloseAllPositionsRequest struct {
 }
 
 // CloseAllPositions liquidates all open positions at market price.
+// It returns the list of orders that were submitted to close the positions.
+// If an error occurs while closing a position, the error will be returned
```

Comment:
In Go it's quite unorthodox to return both an error and something else, however, this is exactly what you're doing here. In understand that this is mainly for backward compatibility, so I'm okay with it, but I would highlight it in the docstring, something like: "If errors occur while closing some of the positions, the errors will also be returned (possibly among orders)".